package.json

```diff
@@ -22,6 +22,7 @@
     "@types/react": "^16.9.0",
     "@types/react-cytoscapejs": "^1.1.0",
     "@types/react-dom": "^16.9.0",
+    "@use-it/event-listener": "^0.1.5",
```

Comment:
As far as this goes, I've built a better one for work from scratch and it's had a couple iterations on it but I obviously can't use that code directly. This package was very close and took in the proper considerations.

https://github.com/donavon/use-event-listener